src/poly/poly.jl

```diff
@@ -945,6 +945,22 @@ function zero!(x::spoly)
    return x
 end
 
+###############################################################################
+#
+#   Changing base ring
+#
+###############################################################################
+
+@doc Markdown.doc"""
+   change_base_ring(p::spoly, C::Any)
+> Return a polynomial ring, whose coefficient ring is subsituted by $C$.
```

Comment:
You should mention what C can be here. I.e. it should be possible to pass a Map, Julia function or Ring object.

Reply:
I just read your conversation for AbstractAlgebra regarding the topic. Do I understand it correctly, that change_base_ring will only take rings as input instead of "Any"? In this case I would change it in Singular to rings only.

Reply:
Hmm, I hope not. Let me check.

Reply:
I read Tommys last comment and I will adapt it.